### PR TITLE
[Smoke test] Force kill all code.exe test instances before test

### PR DESCRIPTION
**Pull request by ruslan-bikkinin**

### CODE REVIEW

File: test/smoke/src/helpers/setupEnvironmentHelper.ts [hunk 1]
```diff
@@ -296,6 +296,13 @@ import * as Icon from '@expo/vector-icons';
         const metroConfigPath = path.join(appPath, "metro.config.js");
         console.log(`*** Patching  ${metroConfigPath}`);
         const patchContent = `
+// To avoid EPERM errors on Windows
```
Comment (SounD120) [inline test/smoke/src/helpers/setupEnvironmentHelper.ts]: Do we have an URL link for this change describing why this change is needed?

Comment (ruslan-bikkinin) [reply test/smoke/src/helpers/setupEnvironmentHelper.ts]: Unfortunately, we haven't one. I'll add more details though.
